Shared/ArticleStyles/ArticleThemePlist.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  ArticleThemePlist.swift
+//  NetNewsWire
+//
+//  Created by Stuart Breckenridge on 19/09/2021.
+//  Copyright © 2021 Ranchero Software. All rights reserved.
+//
+
+import Foundation
+
+public struct ArticleThemePlist: Codable {
+	public var name: String
+	public var themeIdentifier: String
+	public var creatorHomePage: String
+	public var creatorName: String
+	public var version: Int
+	
+	enum CodingKeys: String, CodingKey {
+		case name = "Name"
+		case themeIdentifier = "ThemeIdentifer" // FIXME: Spelling error!
+		case creatorHomePage = "CreatorHomePage"
+		case creatorName = "CreatorName"
+		case version = "Version"
+	}
+}
```

Comment:
If we are going to add this, we should go ahead and change `ArticleTheme` to use it to remove redundant definitions. Also, let's just go ahead and fix the typo.

Reply:
This has been implemented for `ArticleTheme`.